### PR TITLE
feat(profiling): update docs for beta

### PR DESCRIPTION
**Pull request by JonasBa**
Update docs for beta

### CODE REVIEW

File: src/platforms/node/common/profiling/index.mdx [hunk 1]
```diff
@@ -56,4 +56,7 @@ SENTRY_PROFILER_LOGGING_MODE=eager node script.js
 
 We recommend you have your own CPU resource-monitoring in place, because the actual resource use could be environment-dependent.
 
+## Precompiled binaries
```
Comment (lizokm) [inline src/platforms/node/common/profiling/index.mdx:59]:
```suggestion
## Precompiled Binaries
```

File: src/platforms/node/common/profiling/index.mdx [hunk 2]
```diff
@@ -56,4 +56,7 @@ SENTRY_PROFILER_LOGGING_MODE=eager node script.js
...
+Starting from version 0.1.0, @sentry/profiling-node package precompiles binaries for a number of common architectures - this minimizes the tooling required to run the package and in most cases avoids compiling the package from source which speeds up installation. The set of common architectures should cover a wide variety of use cases, but in case that
+
 If you have feedback or experience different behavior, please open an issue on sentry/profiling-node repository.
```
Comment (lizokm) [inline src/platforms/node/common/profiling/index.mdx:62]:
```suggestion
Starting from version 0.1.0, @sentry/profiling-node package precompiles binaries for a number of common architectures. This minimizes the tooling required to run the package and avoids compiling the package from source in most cases, which speeds up installation. The set of common architectures should cover a wide variety of use cases, but if you have feedback or experience different behavior, please open an issue on the sentry/profiling-node repository.
```

**Review by lizokm — APPROVED** [review]
Just a few small tweaks, otherwise, this LGTM

**Comment by JonasBa** [comment]
Thank you so much for the review and suggestions @lizokm, I merged both of them :)

**Pull request merged**